src/main/java/org/ilgcc/app/pdf/ProviderSubmissionFieldPreparerService.java

```diff
@@ -66,12 +66,11 @@ public class ProviderSubmissionFieldPreparerService implements SubmissionFieldPr
     public Map<String, SubmissionField> prepareSubmissionFields(Submission familySubmission, PdfMap pdfMap) {
         Map<String, SubmissionField> results = new HashMap<>();
 
-        // TODO: We were using enableMultipleProviders but we can just use the key from the multiprovider data structure
-        // When enable multi provider in prod, the else of this code can be removed
-        if (familySubmission.getInputData().containsKey("providers")) {
-            Map<String, List<Map<String, Object>>> mergedChildrenAndSchedules =
-                    getRelatedChildrenSchedulesForEachProvider(familySubmission.getInputData());
+        Map<String, List<Map<String, Object>>> mergedChildrenAndSchedules =
+                getRelatedChildrenSchedulesForEachProvider(familySubmission.getInputData());
 
+        // When enable multi provider in prod, the else of this code can be removed
```

Comment:
based off what @spokenbird said to me in a Tuple last week, can we update the comment to include the TODO and also the environment variable name... so when we go to productize it, it's a little easier to find this one

```suggestion
        // TODO: When enable multi provider in prod, the else of this code can be removed when ENABLE_MULTIPLE_PROVIDERS is productized
```